console/src/assets/i18n/ja.json

```
@@ -0,0 +1,2095 @@
+    "STEP": "ステップ",
+    "SETUP": "セットアップ",
+    "COMINGSOON": "近日公開",
+    "TABLE": {
+      "SHOWUSER": "ユーザー {{value}} を表示する"
+    }
+  },
+  "MEMBERROLES": {
+    "IAM_OWNER": "すべての組織を含むインスタンス全体を管理する権限を持ちます",
+    "IAM_OWNER_VIEWER": "すべての組織を含むインスタンス全体を閲覧する権限を持ちます",
+    "IAM_ORG_MANAGER": "組織の作成および管理する権限を持ちます",
+    "IAM_USER_MANAGER": "ユーザーの作成および管理する権限を持ちます",
+    "ORG_OWNER": "組織全体に対する権限を持ちます",
+    "ORG_USER_MANAGER": "組織のユーザーを作成および管理する権限を持ちます",
+    "ORG_OWNER_VIEWER": "組織全体を閲覧する権限を持ちます",
+    "ORG_USER_PERMISSION_EDITOR": "ユーザーグラントを管理する権限を持ちます",
+    "ORG_PROJECT_PERMISSION_EDITOR": "プロジェクトグラントを管理する権限を持ちます",
+    "ORG_PROJECT_CREATOR": "所有するプロジェクトと配下の設定を作成する権限を持ちます",
+    "PROJECT_OWNER": "特定のプロジェクト全体を管理する権限を持ちます",
+    "PROJECT_OWNER_VIEWER": "特定のプロジェクト全体を閲覧する権限を持ちます",
+    "PROJECT_OWNER_GLOBAL": "全てのプロジェクトを管理する権限を持ちます",
+    "PROJECT_OWNER_VIEWER_GLOBAL": "全てのプロジェクトを閲覧する権限を持ちます",
+    "PROJECT_GRANT_OWNER": "プロジェクトグラントを管理する権限を持ちます",
+    "PROJECT_GRANT_OWNER_VIEWER": "プロジェクトグラントを閲覧する権限を持ちます"
+  },
+  "OVERLAYS": {
+    "ORGSWITCHER": {
+      "TEXT": "コンソール内のすべての組織設定とテーブルは、選択した組織に基づいています。このボタンをクリックして組織を切り替えるか、新しい組織を作成してください。"
+    },
+    "INSTANCE": {
+      "TEXT": "インスタンス設定にアクセスするには、ここをクリックしてください。強力なアクセス権限を許可されている場合は、このボタンにのみアクセスできることに注意してください。"
+    },
+    "PROFILE": {
+      "TEXT": "ここでは、ユーザーアカウントを切り替えて、セッションとプロフィールを管理できます。"
+    },
+    "NAV": {
+      "TEXT": "このナビゲーションは、上記の選択した組織またはインスタンスに基づいて変更されます"
+    },
+    "CONTEXTCHANGED": {
+      "TEXT": "注意！ 組織のコンテキストが変更されました。"
+    }
+  },
+  "FILTER": {
+    "TITLE": "絞り込み",
+    "STATE": "ステータス",
+    "DISPLAYNAME": "表示用ユーザー名",
+    "EMAIL": "Eメール",
+    "USERNAME": "ユーザー名",
+    "ORGNAME": "組織名",
+    "PROJECTNAME": "プロジェクト名",
+    "RESOURCEOWNER": "リソース所有者",
+    "METHODS": {
+      "1": "等しい",
+      "5": "含む",
+      "7": "で終わる"
+    }
+  },
+  "KEYBOARDSHORTCUTS": {
+    "TITLE": "キーボードショートカット",
+    "UNDERORGCONTEXT": "組織ページ内",
+    "SIDEWIDE": "サイト全体のショートカット",
+    "SHORTCUTS": {
+      "HOME": "<strong>G</strong>o to <strong>H</strong>ome",
+      "INSTANCE": "<strong>G</strong>o to <strong>I</strong>instance",
+      "ORG": "<strong>G</strong>o to <strong>O</strong>rganization",
+      "ORGSETTINGS": "<strong>G</strong>o to Organization <strong>S</strong>ettings",
+      "ORGSWITCHER": "組織を変更する",
+      "ME": "自分のプロフィールに移動する",
+      "PROJECTS": "<strong>G</strong>o to <strong>P</strong>rojects",
+      "USERS": "<strong>G</strong>o to <strong>U</strong>sers",
+      "USERGRANTS": "<strong>G</strong>o to <strong>A</strong>uthorizations",
+      "ACTIONS": "<strong>G</strong>o to Actions and <strong>F</strong>lows",
+      "DOMAINS": "<strong>G</strong>o to <strong>D</strong>omains"
+    }
+  },
+  "RESOURCEID": "リソースID",
+  "NAME": "名前",
+  "VERSION": "バージョン",
+  "TABLE": {
+    "NOROWS": "データなし"
+  },
+  "ERRORS": {
+    "REQUIRED": "一部の必須項目が不足しています。",
+    "TOKENINVALID": {
+      "TITLE": "トークンが期限切れになりました。",
+      "DESCRIPTION": "下のボタンをクリックして、もう一度ログインする。"
+    }
```

```suggestion
    },
    "INVALID_FORMAT": "The formatting is invalid.",
    "NOTANEMAIL": "The given value is not an e-mail address.",
    "MINLENGTH": "Must be at least {{requiredLength}} characters long.",
    "UPPERCASEMISSING": "Must include an uppercase character.",
    "LOWERCASEMISSING": "Must include a lowercase character.",
    "SYMBOLERROR": "Must include a symbol or punctuation mark.",
    "NUMBERERROR": "Must include a digit.",
    "PWNOTEQUAL": "The passwords provided do not match.",
    "PHONE": "The phone number must start with 00 or +."
```

we had some new translations in the meantime.. could you set the correct values.. Thx